### ISSUE TITLE
"zwe components enable" is not working for the extensions

### ISSUE THREAD

**Issue by arxioly** [zowe/zowe-install-packaging#2827]
**Describe the bug**
The enabling of the extension as part of the `zwe components install` command or standalone `zwe components enable` is returning the error `Error ZWEL0152E: Cannot find component broadcom-idt.` 
After some investigation the reason for that could be that `ZWE_zowe_extensionDirectory` parameter is not set, though it is set `zowe.extensionDirectory` in zowe.yaml.

**To Reproduce**
1. Run the command `zwe components install -o "/<path-to-extension-package>/extension-package.zip" -c "<path-to-zowe-yaml>"`
2. See the log. Something like this will be printed:
```
Install extension-package.zip
Tagging files for ISO8859-1
Tagging files for UTF-8 as binary
Tagging files for binary
Converting ebcdic files
Tagging files for IBM-1047
Marking scripts as executable
Error ZWEL0152E: Cannot find component broadcom-idt.
```

**Expected behavior**
The extension added and enabled in zowe.yaml

**Logs**
Part of log from run of this command:
 `export PATH=$PATH:<path-to-zowe>/v2.0.0/bin && export NODE_HOME=<path-to-nodejs> && export ZWE_CLI_PARAMETER_CONFIG=<path-to-zowe-yaml> && zwe components enable --component-name broadcom-idt`
```log
+ [ -z /zowe/zowe.yaml ]
+ [ ! -f /zowe/zowe.yaml ]
+ + find_component_directory broadcom-idt
+ 1<TMP> /tmp/shEhbfegdgA
+ component_id=broadcom-idt
+ component_dir=
+ [ -d /zowe/v2.0.0/components/broadcom-idt ]
+ [ -n  ]
+ [ -n  ]
+ echo
component_dir=
+ [ -z  ]
+ print_error_and_exit Error ZWEL0152E: Cannot find component broadcom-idt.  152
``` 

**Environment:**
 - OS: z/OS 2.5
 - Zowe version: 2.0.0

**Comment by jackjia-ibm** [comment]
I think we have a fix for this https://github.com/zowe/zowe-install-packaging/pull/2769 already, but with some discussions this fix wasn't confirmed and made into GA release. The discussion was in zowe-release channle https://openmainframeproject.slack.com/archives/G01M2C8QB3K/p1649707949819189.

I will make sure this will be merged in v2.1.

The mitigation for now is manually set

```
components:
  broadcom-idt:
    enabled: true
```

in your zowe.yaml. This is expected result comes from `zwe components enable` command.

**Comment by std4lqi** [comment]
We just added `manifest.yaml` under the directory of Zowe v1 app plugin and got the same `Error ZWEL0152E: Cannot find component sample-trial-app`  from `zwe components install -c zowe.yaml -o /zowe/instance/instance2.v2/trial-app`. 
```
name: sample-trial-app
id: org.zowe.sample-trial-app
title: Sample Trial App
description: A sample application for Zowe
appfwPlugins:
  - path: .
```

Any guideline to upgrade Zowe v1 app plugin to component?  I'm not sure whether we did it right.

**Comment by jackjia-ibm** [comment]
@std4lqi Quick look on your `manifest.yaml` looks correct. The issue is not critical which only affects updating `zowe.yaml` to enable the newly installed component. The component itself should have been installed correctly. The only thing to enable  the component (to bypass the issue for now) is adding `components.sample-trial-app.enabled: true` in your `zowe.yaml`.

To define manifest.yaml and packaging for v2, https://docs.zowe.org/stable/extend/packaging-zos-extensions is still the best doc I can think of. It also has a link to full manifest.yaml reference. Please let me know if anywhere are not clear/confusing or can be improved. Thanks.

**Comment by jackjia-ibm** [comment]
Please re-open the issue if the issue is not fixed in v2.1.0 release. Thanks.
